wiki/Static-Analysis-Checks.md

```diff
@@ -138,8 +141,10 @@ exempted_file_path: "app/src/main/java/org/oppia/android/app/activity/ActivityCo
 2. Add an explanation to your PR description detailing why this exemption is correct.
 
 Following are the cases where its valid to have test file exemptions:
-1. Interface files
-2. Files with only constants defined (no logic)
+1. Interface files.
+2. Files with only constants defined (no logic).
+3. Cases when tests fail to pass code coverage.
+4. Cases when a test is verifying additional functionality beyond what can be tested in its usual test suite (for these cases use the 'is_extra_test' property in the exemption structure).
```

Comment:
It might be worth adding an example here as well. It took me a while to understand why the math parser tests were exempted.

Reply:
Good point--added an example and made some other small clean-ups in this file.